site/dat/repo/blazemeter.json

```diff
@@ -746,11 +746,8 @@
       "co.signal.mqmeter.MQClientSampler"
     ],
     "versions": {
-      "2.1.0": {
-        "downloadUrl": "https://github.com/JoseLuisSR/mqmeter/releases/download/2.1.0/mqmeter-2.1.0.jar",
-        "libs": {
-          "com.ibm.mq": "https://search.maven.org/remotecontent?filepath=com/ibm/mq/com.ibm.mq.allclient/9.0.4.0/com.ibm.mq.allclient-9.0.4.0.jar"
-        }
+      "2.1.1": {
```

Comment:
When you release new version, please add it into "versions" instead of replacing. This way, users with old version will get nicer upgrade experience. Also, some users might want to downgrade plugin in case of incompatible changes, so having old version is very important.